src/libraries/System.Runtime/tests/System.Globalization.Tests/CultureInfo/CultureInfoCurrentCulture.cs

```diff
@@ -154,9 +152,7 @@ public void CurrentCulture_BasedOnLangEnvVar(string langEnvVar, string expectedC
         public void CurrentCulture_DefaultWithNoLang(string langEnvVar)
         {
             var psi = new ProcessStartInfo();
-            psi.Environment.Clear();
-
-            CopyEssentialTestEnvironment(psi.Environment);
+            TestEnvironment.ClearGlobalizationEnvironmentVars(psi.Environment);
 
```

Comment:
Does it make sense to keep this but move before `TestEnvironment.ClearGlobalizationEnvironmentVars(psi.Environment);`. I think this was added for a specific reason? 

[https://github.com/dotnet/corefx/pull/11106/files](https://github.com/dotnet/corefx/pull/11106/files)

Reply:
The issue that was fixing is https://github.com/dotnet/runtime/issues/18281

Since we're no longer clearing the environment any LD_LIBRARY_PATH in the parent process should propagate to the new process.